File: 백준/Bronze/1159. 농구 경기/농구 경기.c
#include <stdio.h>
#define SIZE 26

int main() {
	int num, i, count[SIZE] = { 0 }, flag = 0;
	char name[31];

	scanf("%d", &num);

	for (i = 0; i < num; i++) {
		scanf("%s", name);
		count[name[0] - 'a']++;
	}

	for (i = 0; i < SIZE; i++) {
		if (count[i] >= 5) {
			if (!flag)
				flag = 1;
			printf("%c", 'a' + i);
		}
	}
	if (!flag)
		printf("PREDAJA");

	printf("\n");
}

File: 백준/Bronze/1159. 농구 경기/농구 경기.py
import sys

input = sys.stdin.readline

answer = []
n = int(input())
name = []
count = [0] * 26

for _ in range(n):
    name.append(input().rstrip())

for x in name:
    count[ord(x[0])-ord('a')] += 1

for i in range(26):
    if count[i] >= 5:
        answer.append(chr(ord('a')+i))

print("".join(answer) if answer else "PREDAJA")